pallets/living-assets-ownership/src/lib.rs

```diff
@@ -32,17 +41,17 @@ pub mod pallet {
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 	}
 
-	/// Mapping from collection id to owner
-	#[pallet::storage]
-	#[pallet::getter(fn owner_of_collection)]
-	pub(super) type OwnerOfCollection<T: Config> =
-		StorageMap<_, Blake2_128Concat, CollectionId, T::AccountId, OptionQuery>;
-
 	/// Collection counter
 	#[pallet::storage]
 	#[pallet::getter(fn collection_counter)]
 	pub(super) type CollectionCounter<T: Config> = StorageValue<_, CollectionId, ValueQuery>;
```

Comment:
What about use [frame_support::storage::types::CountedStorageMap](https://paritytech.github.io/substrate/master/frame_support/storage/types/struct.CountedStorageMap.html) instead?

Reply:
Nice helper struct, however: 
In our system, the CollectionId is of type `u64`, which means that the maximum number of collections we can potentially handle is `2^64`.
On the other hand, `CountedStorageMap` uses a counter of type `u32` to keep track of the number of items in the map.